Dockerfile

```diff
@@ -1,17 +1,45 @@
-FROM node:16.14-alpine3.14
```

Comment:
We upgraded node to current LTS and created the `builder` and `deps` stages.
In `builder` we install all dependencies (dev dependencies as well since we need babel) and run the babel build of our source code.
In `deps` we install all production dependencies.

In the production stage we copy the source code built with babel on `builder` and the installed dependencies from `deps`.
We also need to install `dumb-init` since it is our process supervisor.